frontend/integration-tests/protractor.conf.ts

```diff
@@ -293,3 +293,18 @@ export const retry = async <T>(
     return retry(fn, retries - 1, interval * 2);
   }
 };
+
+export async function fillInput(elem: any, value: string) {
```

Comment:
We should update the kubevirt tests to use this as well to avoid having it implemented in two places.

Reply:
Ok, I was hesitant to change the kubevirt package code.  Will do

Reply:
@dtaylor113 yeah, it probably should

Reply:
Ok, refactored and updated.